website/docs/migrations/11-12.md

```diff
@@ -218,3 +218,46 @@ const projectConfig = {
 	},
 };
 ```
+
+### Update ES Lint config
+
+**Migration time: ~10min.**
+
+Depending on the project current state you might need to account for time spent fixing errors and warnings produced by the new ES Lint settings so the migration time may vary.
+
+ES Lint configuration has been updated. To make sure everything works with the new version update the project node version to 20 or higher and update the ES Lint configuration. The steps are described below.
+
+1. Delete the `.eslintrc.js`  and `.eslintignore` files from the theme root.
+
+2. Add `eslint.config.mjs` file to the theme root with the following content:
+```js
+import config from '@eightshift/frontend-libs/linters/eslint.config.mjs';
+
+export default [{
+    ignores: ["public/**/*.js"],
+}, ...config];
+```
+
+3. Install the required dependencies:
+```bash
+npm install @eslint/js -D
+npm install @eslint/eslintrc -D
+```
+
+4. Add `.swcrc` to theme root with the following content:
+```json
+{
+	"jsc": {
+		"parser": {
+			"syntax": "ecmascript",
+			"jsx": true,
+			"decorators": false,
+			"dynamicImport": true
+		}
+	}
+}
+```
+
+5. Make sure the project is using `node` version 20 or higher. If you are deploying with GitHub actions you will need to update to `node: [22.x]` version in deploy config files:\
```

Comment:
deploy process is not the part of the generic 8shift setup and any one can use this as is. You can leave the part for the node version but remove all the mentions of the deploy process